Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:17.04
```

Comment:
Preferably, use the `alpine` image, there's a Python 3.6 version available. It produces a much smaller image, allowing new people to get started faster.